cpp/perspective/src/include/perspective/scalar.h

```diff
@@ -207,49 +207,49 @@ template <>
 t_tscalar t_tscalar::coerce_numeric<bool>() const;
 
 template <>
-std::int64_t t_tscalar::get() const;
+PERSPECTIVE_EXPORT std::int64_t t_tscalar::get() const;

```

Comment:
@sc1f i assume all these specializations are necessary?

Reply:
Yeah, they all need to be explicitly specialized.